src/redux/sagas/extensions/extensionInstallAndEnable.ts

```diff
@@ -109,8 +115,14 @@ export function* extensionInstallAndEnable({
       yield takeFrom(setUserRoles.channel, ActionTypes.TRANSACTION_CREATED);
     }
 
-    yield initiateTransaction(installExtension.id);
-    yield waitForTxResult(installExtension.channel);
+    try {
+      yield initiateTransaction(installExtension.id);
+      yield waitForTxResult(installExtension.channel);
+    } catch (error) {
+      // Declare an error step for the error handler
+      error.step = ExtensionInstallAndEnableErrorStep.InstallExtension;
```

Comment:
we could throw specific errors and then have typeguards too
Like for example a `ExtensionInstallError` which has a step property like this component I just randomly pulled up
![image](https://github.com/user-attachments/assets/af490364-54b5-4cf5-a2ad-a2af5ba50b7b)

and then we can check the error like this:
`if (error instanceof ExtensionInstallError) `

and the step property will be defined

Maybe it's overkill, just wanted to drop this here :D